CHANGELOG.md

```diff
@@ -3,6 +3,15 @@
 
 # Rivet Changelog
 
+## 0.21.1
+
+> Released 24 May 2023
+
+### Fixed
+
+* Pipe output to null when calling `Connect-Database` function from within `Connect-RivetSession` function.

```

Comment:
This is an implementation detail that means nothing to the user. Changelogs should instead convey how the software has changed from the user's perspective.

> Fixed: `Connect-RivetSession` no longer returns errant boolean values.